packages/rest/src/keys.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/rest
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {CoreBindings} from '@loopback/core';
+
+export namespace RestBindings {
+  // RestServer-specific bindings
+  export const CONFIG = `components.RestServer.config`;
+  export const PORT = 'http.port';
+  export const HANDLER = 'http.handler';
+
+  export const API_SPEC = 'components.RestServer.apiSpec';
+  export const SEQUENCE = 'sequence';
+
+  export namespace SequenceActions {
+    export const FIND_ROUTE = 'sequence.actions.findRoute';
+    export const PARSE_PARAMS = 'sequence.actions.parseParams';
+    export const INVOKE_METHOD = 'sequence.actions.invokeMethod';
+    export const LOG_ERROR = 'sequence.actions.logError';
+    export const SEND = 'sequence.actions.send';
+    export const REJECT = 'sequence.actions.reject';
```

Comment:
Ditto, I think all of these action keys should be prefixed with `rest.` or similar to distinguish from sequence actions for other transports.